pinot-spi/src/main/java/org/apache/pinot/spi/data/Schema.java

```diff
@@ -526,6 +544,14 @@ public SchemaBuilder() {
       _schema = new Schema();
     }
 
+    public static SchemaBuilder copyWithoutFields(Schema other) {
```

Comment:
Is this needed?

Reply:
No. I was planning to use this method to fix the clone method, but it wasn't trivial (and it was not related to this PR).

I'm removing this method